dev/ci/ci-coqtail.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+FORCE_GIT=1
+git_download coqtail
+python3 -m pip install pytest
```

Comment:
I think we may prefer to do this in the docker image.

Reply:
I tried adding a line to dev/ci/docker/bionic_coq/Dockerfile but it didn't seem to work. Is there somewhere else I should do it?

Reply:
It needs admin action, now running.